File: src/main/frontend/src/app/common/position.ts
export class Position {

  constructor(private _position: String, private _startDate: String, private _endDate: String) {
  }

  get position(): String {
    return this._position;
  }

  set position(value: String) {
    this._position = value;
  }

  get startDate(): String {
    return this._startDate;
  }

  set startDate(value: String) {
    this._startDate = value;
  }

  get endDate(): String {
    return this._endDate;
  }

  set endDate(value: String) {
    this._endDate = value;
  }
}

File: src/main/java/org/_3rev/resume/valueobject/Position.java
package org._3rev.resume.valueobject;

public class Position {

    private String position;
    private String startDate;
    private String endDate;

    public Position() {
        super();
    }

    public Position(String position, String startDate, String endDate) {
        this.position = position;
        this.startDate = startDate;
        this.endDate = endDate;
    }

    public String getPosition() {
        return position;
    }

    public void setPosition(String position) {
        this.position = position;
    }

    public String getStartDate() {
        return startDate;
    }

    public void setStartDate(String startDate) {
        this.startDate = startDate;
    }

    public String getEndDate() {
        return endDate;
    }

    public void setEndDate(String endDate) {
        this.endDate = endDate;
    }
}
